java-core/google-cloud-core/src/test/java/com/google/cloud/TimestampTest.java

```diff
@@ -157,67 +154,67 @@ public void toFromSqlTimestamp() {
   }
 
   @Test
-  public void boundsSecondsMin() {
+  void boundsSecondsMin() {
     try {
       Timestamp.ofTimeSecondsAndNanos(Timestamp.MIN_VALUE.getSeconds() - 1, 999999999);
-      Assert.fail();
+      Assertions.fail();
     } catch (IllegalArgumentException ex) {
       assertNotNull(ex.getMessage());
     }
   }
 
   @Test
-  public void boundsSecondsMax() {
+  void boundsSecondsMax() {
     try {
       Timestamp.ofTimeSecondsAndNanos(Timestamp.MAX_VALUE.getSeconds() + 1, 0);
-      Assert.fail();
+      Assertions.fail();
     } catch (IllegalArgumentException ex) {
       assertNotNull(ex.getMessage());
     }
   }
 
   @Test
-  public void boundsNanosMin() {
+  void boundsNanosMin() {
     try {
       Timestamp.ofTimeSecondsAndNanos(TEST_TIME_SECONDS, -1);
-      Assert.fail();
+      Assertions.fail();
     } catch (IllegalArgumentException ex) {
       assertNotNull(ex.getMessage());
     }
   }
 
   @Test
-  public void boundsNanosMax() {
+  void boundsNanosMax() {
```

Comment:
Perhaps these can be converted to assertThrows()?

Reply:
Done